corehq/apps/reminders/tests/test_responsiveness.py

```diff
@@ -22,8 +22,8 @@ def setUp(self):
 
     def get_reminders(self):
         return CaseReminder.view('reminders/by_domain_handler_case',
-            start_key=[self.domain],
```

Comment:
How did this ever work?

Reply:
Excellent question.  Cloudant enforces that the kwarg be ```startkey```, maybe restkit is more permissive in some fashion?  Weird AF.